packages/format-library/src/link/inline.js

```diff
@@ -222,7 +236,12 @@ class InlineLinkUI extends Component {
 
 		return (
 			<PositionedAtSelection
-				key={ `${ value.start }${ value.end }` /* Used to force rerender on selection change */ }
```

Comment:
Possible dumb question: Couldn't a minimal solution just be to remove `key`?

Reply:
What if you move the caret from one link to another?

Reply:
> What if you move the caret from one link to another?

That's true. I wonder then if it could either act more like the `Autocomplete` component in [rendering `Popover` with a `getAnchorRect`](https://github.com/WordPress/gutenberg/blob/0b93ad47797a1ba2cea3c83a7f973d33abcd351f/packages/components/src/autocomplete/index.js#L469) that updates with the selection (but keeps the same element so not to unmount and re-trigger animation), or if `PositionedAtSelection` should update its styling in response to selection changes rather than [once at mount](https://github.com/WordPress/gutenberg/blob/0b93ad47797a1ba2cea3c83a7f973d33abcd351f/packages/components/src/positioned-at-selection/index.js#L50-L52).

I'm not totally sure the ramifications of either of these alternatives. Just trying to consider in terms of the least potential for impact.